arm/Microsoft.KubernetesConfiguration/extensions/deploy.bicep

```
@@ -1,10 +1,10 @@
-@description('Required. The name of the Flux Configuration')
+@description('Required. The name of the Flux Configuration.')
 param name string
 
 @description('Optional. Enable telemetry via the Customer Usage Attribution ID (GUID).')
 param enableDefaultTelemetry bool = true
 
-@description('Required. The name of the AKS cluster that should be configured.')
+@description('Conditional. The name of the AKS cluster that should be configured. Required if the template is used in a standalone deployment.')
```

This is a separate provider so technically not an AKS child resource. This change should probably be rolled back.
```suggestion
@description('Required. The name of the AKS cluster that should be configured.')
```

---

done